### PR TITLE
When editing issue only update changed properties

### LINKED ISSUE #641: Editing only issue labels/milestones displays edited text in header
spectacle zu3326

The underlined text is what appears on issues even though I didn't modify issue's body or title. It doesn't appear when adding labels from the web interface.

### PR DESCRIPTION
**Pull request by Tunous**
Closes #641

### CODE REVIEW

File: github-api/src/main/java/org/eclipse/egit/github/core/service/IssueService.java [hunk 1]
```diff
@@ -47,7 +30,24 @@
...
+import static org.eclipse.egit.github.core.client.IGitHubConstants.CHARSET_UTF8;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_COMMENTS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_EVENTS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_ISSUES;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_LEGACY;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_REACTIONS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_REPOS;
+import static org.eclipse.egit.github.core.client.IGitHubConstants.SEGMENT_SEARCH;
+import static org.eclipse.egit.github.core.client.PagedRequest.PAGE_FIRST;
+import static org.eclipse.egit.github.core.client.PagedRequest.PAGE_SIZE;
```
Comment (maniac103) [inline github-api/src/main/java/org/eclipse/egit/github/core/service/IssueService.java]: Reordering seems unneeded (yes, I agree it's annoying, I just want to keep the diff to the original bindings as small as possible).

Comment (Tunous) [reply github-api/src/main/java/org/eclipse/egit/github/core/service/IssueService.java]: Every single time before I commit I view whole diff and check if there is anything that I've missed, yet I still manage to always get these import changes unnoticed.

I probably have a checkbox to auto organize them enabled in a commit dialog... Will check and fix that later 😆

(I wish there was an option to use different styling rules per module in AS)

Comment (Tunous) [reply github-api/src/main/java/org/eclipse/egit/github/core/service/IssueService.java]: Fixed and disabled the checkbox so I shouldn't make that mistake again.

**Comment by maniac103** [comment]
Thanks for the fixup :)

**Pull request merged**